packages/suite/src/views/dashboard/components/AssetsCard/components/AssetGrid/index.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Network } from '@wallet-types';
+import { CoinLogo, Icon, variables } from '@trezor/components';
+import {
+    FiatValue,
+    SkeletonCircle,
+    SkeletonRectangle,
+    Ticker,
+    Translation,
+} from '@suite-components';
+import { CoinBalance } from '@wallet-components';
+import { isTestnet } from '@wallet-utils/accountUtils';
+import * as routerActions from '@suite-actions/routerActions';
+import { useActions, useAccountSearch, useLoadingSkeleton } from '@suite-hooks';
+
+const Col = styled.div`
+    display: flex;
+    align-items: center;
+    padding: 16px 0px;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    padding-right: 20px;
+`;
+
+const CoinNameWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    cursor: pointer;
+`;
+
+const UpperRowWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    width: 302px;
+    padding: 0px 15px 15px 15px;
+    border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
+`;
+
+const Coin = styled.div`
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    align-items: center;
+`;
+
+const LogoWrapper = styled.div`
+    padding-right: 12px;
+    display: flex;
+    align-items: center;
+`;
+
+const CoinGridWrapper = styled.div`
+    width: 330px;
+    height: 118px;
+    margin: 0px 9px 18px 0px;
+    padding: 19px 16px 18px 12px;
+    border-radius: 6px;
+    background-color: #ffffff;
```

Comment:
You need to pick a color from `props.theme` otherwise the dark mode will show white background which looks ugly.
<img width="1000" alt="Screenshot 2021-04-08 at 10 08 22" src="https://user-images.githubusercontent.com/6961901/113991065-625e0080-9852-11eb-8517-1090803a37eb.png">

Reply:
Done